standards/certification/getting-scs-compatible-certified.md

```diff
@@ -0,0 +1,114 @@
+that lets their applications run in a reliable way.
+
+The SCS certification process typically consists of a few simple steps:
+
+1. Running the SCS compliance test suite and adjusting the infrastructure until it passes.
+2. Any additional declarations (for non-testable aspects) are written and passed to the SCS Forum.
+3. The operator must be a member ("shaper" or "advisor" level) of the Forum SCS-Standards in the
+   OSB Alliance (a non-profit) and pay the respective membership fees. Alternatively fees can
+   be paid without becoming a member.
+4. The cloud can be listed on the SCS pages as SCS-compatible with a compatibility status that is
+   updated on a daily basis. SCS then tests the infrastructure on a daily basis.
+
+## Self-testing and technical adjustments
+
+In order for a cloud service offering to obtain a certificate, it has to
+conform to all standards of the respective scope, which will be tested at
+regular intervals, and the results of these tests will be made available
+publicly. For more details on how to become certified, please consult the
+corresponding [document](/standards/scs-0004-v1-achieving-certification).
+
+The best approach to get your cloud into compliance is by installing the
+test suite locally. Have a look at the
+[example](/standards/certification/test-and-adapt-example).
+
+A description how SCS-compatible IaaS compliance can be achieved on environments that use different
+OpenStack implementations is written up in a blog article
+[Cost of making an OpenStack Cluster SCS compliant](https://scs.community/2024/05/13/cost-of-making-an-openstack-cluster-scs-compliant/).
+
+## Declarations
+
+For the SCS-compatible IaaS v5 standard, the providers must -- if they implement availability zones
+at all (which is optional) -- guarantee certain levels of independence for these. This can not
+be fully tested by an automated test. The process thus envisions that providers must create some
+documentation on the physical infrastructure and how it maps to availability zones and declare that
+this documentation reflects the truth. SCS will review the docs and judge whether they meet the
+criteria. In case of doubt, audits can be performed.
+
+## Forum SCS-Standards @ OSBA
+
+The SCS brand belongs to the Open Source Business Alliance e.V. (OSBA), an non-profit organization and
+association for the Open Source Industry in Germany. The OSBA creates the Forum SCS-Standards
+which performs the work to evolve the SCS standards, develops the tests and perform the certification
+process.
+
+Members of the OSBA can become also member of the Forum SCS-Standards for an additional membership
+fee, providing the financial resources for the Forum to do its work. Membership in the OSBA is
+open to any organization that supports the goals of the OSBA.
+Alternatively, a certification fee can be paid without any membership; the fee corresponds to the
+lower tier membership fee.
```

Comment:
Please don't place infos at arbitrary locations that might then be outdated. I think, it makes sense to point here to the forum but I would refrain from making statements regarding fees since this will lead to be errors on the long run.